alternator/executor.cc

```diff
@@ -1066,6 +1068,11 @@ static future<executor::request_return_type> create_table_on_shard0(tracing::tra
                     cql3::util::maybe_quote(view_range_key));
             }
             where_clauses.push_back(std::move(where_clause));
+            // LSIs have no tags, but Scylla's "synchronous_updates" feature
+            // (which an LSIs need), is actually implemented as a tag so we
+            // need to add it here:
+             std::map<sstring, sstring> tags_map = {{db::SYNCHRONOUS_VIEW_UPDATES_TAG_KEY, "true"}};
+            view_builder.add_extension(db::tags_extension::NAME, ::make_shared<db::tags_extension>(tags_map));
```

Comment:
is tags_map copied here? maybe move or construct in place

Reply:
The tags_extension constructor takes a `const std::map<sstring, sstring>& tags` so it copies, indeed. I'll try to remember to follow this up with a fix for this class and all its callers to avoid this copy, although I think this is completely negligible - we're talking here about one of the slowest operations in Scylla - creating a table with a view (it's even worse in DynamoDB - taking many seconds), copying an in-memory map with one item isn't a large part of it.